soundata/datasets/urbansound8k.py

```diff
@@ -3,7 +3,9 @@
 .. admonition:: Dataset Info
     :class: dropdown
 
-    *Created By:*
+    **UrbanSound8K**
+
+    **Created By:**
         | Justin Salamon*^, Christopher Jacoby* and Juan Pablo Bello*
```

Comment:
Remove `|` as in [dcase23_task4b.py](https://github.com/soundata/soundata/pull/165/files#diff-ea6737eb033c8c11fd6da408f6e73877aad3dbad484c1079a466d4dc6b398707)